nautobot/extras/views.py

```diff
@@ -1984,42 +1984,32 @@ def destroy(self, request, *args, **kwargs):
         return super().destroy(request, *args, **kwargs)
 
 
-class ScheduledJobListView(generic.ObjectListView):
+class ScheduledJobUIViewSet(
+    mixins.ObjectDetailViewMixin,
+    mixins.ObjectListViewMixin,
+    mixins.ObjectEditViewMixin,
+    mixins.ObjectDestroyViewMixin,
+    mixins.ObjectBulkDestroyViewMixin,
+    mixins.ObjectBulkCreateViewMixin,
+    mixins.ObjectChangeLogViewMixin,
+    mixins.ObjectNotesViewMixin,
```

Comment:
Actually, I don't think `Edit`, `BulkCreate` apply to ScheduledJobs at present, and as `ScheduledJob` is a `BaseModel` only, I don't think `ChangeLog` or `Notes` apply either?